beacon-chain/db/kv/finalized_block_roots.go

```diff
@@ -81,10 +81,18 @@ func (s *Store) updateFinalizedBlockRoots(ctx context.Context, tx *bolt.Tx, chec
 		if i == len(finalized)-1 {
 			// We don't know the finalized child of the new finalized checkpoint.
 			// It will be filled out in the next function call.
-			container = &ethpb.FinalizedBlockRootContainer{}
+			container = &ethpb.FinalizedBlockRootContainer{
+				ParentRoot: finalized[i-1],
+			}
+		} else if i == 0 {
```

Comment:
might be better to use a switch statement ? If not handling `i` in a progressive order is reads better
```
if i == 0 {
// do something
else if i == len(finalized) -1 {
// do something
} else {
// ...
}
```